src/api/baseapi.cpp

```diff
@@ -59,6 +59,7 @@
 #include "tesseractclass.h"  // for Tesseract
 #include "tprintf.h"         // for tprintf
 #include "werd.h"            // for WERD, WERD_IT, W_FUZZY_NON, W_FUZZY_SP
+#include "tabletransfer.hpp"   // for detected tables from tablefind.h
```

Comment:
I suggest to rename that new file to `tabletransfer.h` as that is the standard extension used for Tesseract code.

Please add the file also to `Makefile.am` (sorted by directory and alphabetically).

Reply:
Done as you wished.